services/horizon/internal/init_web.go

```diff
@@ -103,9 +103,14 @@ func initWebActions(app *App) {
 			r.Get("/offers", OffersByAccountAction{}.Handle)
 			r.Get("/trades", TradeIndexAction{}.Handle)
 			r.Get("/data/{key}", DataShowAction{}.Handle)
+			// Linked-Accounts actions:
+			r.Get("/aggregate_balance", AggregateBalanceAction{}.Handle)
+			r.Get("/controlled_balances", ControlledBalancesAction{}.Handle)
```

Comment:
The current horizon version uses underscore in an endpoint. if they can, surely we can too. refer to https://github.com/stellar/go/blob/009e6e45c9f514e8e4f102461fbaefdf92b4f448/services/horizon/internal/web.go#L179

Reply:
from what I gather, "id" is used to identify the page returned. in some cases, its the account id but in others, it might a tx hash or another identifier for the page. In other words, I'll need to use account_id instead of id.